src/content/ui-api/management-api/authentication/index.md

```diff
@@ -0,0 +1,117 @@
+
+![Login helper screenshot](login-utility-results.png)
+
+Here you can inspect the details that will be passed to the management API as part if the ID token. You can use this to verify the details coming from your identity provider, especially the `email` (which is used as your user identifier) and `groups` claim.
+
+The rest of the page helps you set up `kubectl` manually.
+
+## Authenticating for programmatic access {#service-auth}
+
+For programmatic access, for example from CI/CD pipelines, you should not rely on the above authentication mechanism. Instead, please use [service accounts](https://kubernetes.io/docs/reference/access-authn-authz/service-accounts-admin/).
+
+Each Giant Swarm installation provides a service account named `automation` in the `default` namespace. The according token can be found by inspecting this `ServiceAccount` resource. It is usually named starting with `automation-token-`.
+
+**Note:** Since the above token is extremely powerful, **we strongly recommend to create a specific service account for each integration**, binding it to specific roles and only to the required namespaces.
+
+## Single sign-on requirements {#sso-requirements}
+
+Giant Swarm assists all customers with setting up single sign-on (SSO) for the management API. During that process, we make sure that these requirements are met:
+
+- As a customer, you need to decide for an **identity provider** to use. Most enterprise-grade organizations have their solution in place. Since we use [Dex](https://github.com/dexidp/dex) as a connector between the management API and your identity provider, we can support a variety of common standards like OpenID Connect (OIDC) and LDAP.
+
+    For cases where no suited identity provider is available, or not yet available, we recommend to use Github, where an organization and teams can be set up and managed easily.
+
+- Your identity provider must define a **group to be considered as admins** for the Giant Swarm installation. All members of this group automatically get administrative permissions when authenticating with the management API.
+
+- TODO: email claim.
+
+- All additional (non-admin) users you'll want to grant access to the management API in the future will also need to have a user account in your selected identity provider.
+
+## Technical details {#technical-details}
+
+Here we provide some additional details you might want to be aware of, either as a user or as an admin.
+
+### ID-token lifetime {#id-token-ttl}
+
+By default, ID tokens for the management API are issued with a lifetime of **{{% mapi_oidc_token_ttl_minutes %}} minutes**.
+
+When assigning users to groups in your identity provider, and when removing users from groups, it can take up to that {{% mapi_oidc_token_ttl_minutes %}} minutes until the change becomes effective for end users. If a user has authenticated and obtained an ID token before the change, tools like `kubectl` will use that token until it expires.
```

Comment:
```suggestion
When assigning users to groups in your identity provider, and when removing users from groups, it can take up to  {{% mapi_oidc_token_ttl_minutes %}} minutes until the change becomes effective for end users. If a user has authenticated and obtained an ID token before the change, tools like `kubectl` will use that token until it expires.
```